srv/modules/runners/ready.py

```diff
@@ -1 +0,0 @@
-../utils/ready.py
```

Comment:
The 'ready.py' runner still needs to be a 'utils' module as we need to call it in other runners.

In another commit I symlinked the utils module to also act as a runner. If we don't do that, we hit the '__pycache__' issue again.